ansible_base/lib/abstract_models/common.py

```diff
@@ -141,7 +141,7 @@ def get_summary_fields(self):
         for field in self._meta.fields:
             if isinstance(field, models.ForeignObject) and getattr(self, field.name):
                 # ignore relations on inherited django models
-                if field.name.endswith("_ptr"):
+                if field.name.endswith("_ptr") or (field.name in self.ignore_relations):
```

Comment:
@relrod @john-westcott-iv this is a generic DAB base model change, so I am pinging you for a review here. This was discovered troubleshooting the logs & stuff for the RBAC views. The `ignore_relations` will avoid listing things in "related", but it will still show them in "summary_fields".

Reply:
https://github.com/ansible/django-ansible-base/pull/303 I will push a commit to remove it here.